.github/workflows/build-and-deploy.yml

```diff
@@ -141,35 +141,3 @@ jobs:
           docker_image: ${{ needs.docker.outputs.docker_image }}
           azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
           arm-access-key: ${{ secrets.ARM_ACCESS_KEY }}
```

Comment:
also need a database-backup workflow for aks when aks-prod goes live. Either in this PR, or separate.